package.json

```diff
@@ -2,8 +2,14 @@
   "name": "leaflet.functionaltilelayer",
   "version": "0.0.0",
   "description": "Leaflet tile layer with functionally defined URL and support for promises.",
-  "keywords": ["leaflet"],
-  "main": "src/leaflet.functionaltilelayer.js",
+  "keywords": [
+    "leaflet"
+  ],
+  "main": "./src/leaflet.functionaltilelayer.js",
+  "types": "./src/leaflet.functionaltilelayer.d.ts",
   "author": "Ishmael Smyrnow",
-  "license": "MIT"
+  "license": "MIT",
+  "dependencies": {
+    "@types/leaflet": "^1.0.34"
```

Comment:
Is this dependency necessary for people just using the plugin? Shouldn't it be a `devDependency`?

I want to be sensitive to developers who aren't using TypeScript, which is still the majority of developers. I also don't use TypeScript myself, so I'm not sure what the best route to take is.